### PR TITLE
" Your name " screening event article

### PR DESCRIPTION
**Pull request by Aryansinghparmar**

### CODE REVIEW

File: templates/submissions.njk
Comment (wildcraft958) [inline templates/submissions.njk:1]: why validator is here??

File: npm-shrinkwrap.json
Comment (wildcraft958) [inline npm-shrinkwrap.json:1]: @ItsAnkan @destryptor  help!!

File: package.json
Comment (ItsAnkan) [inline package.json:1]: shouldn't be here

File: templates/submissions.njk
Comment (ItsAnkan) [inline templates/submissions.njk:1]: remove validators

**Review by ItsAnkan — CHANGES_REQUESTED** [review]

**Review by ItsAnkan — CHANGES_REQUESTED** [review]
I told you to remove the changes from package.json, not remove package.json ;-;

**Review by ItsAnkan — APPROVED** [review]
Approved

**Pull request merged**